gateways/ssh_tunnel.py

```diff
@@ -19,14 +19,14 @@ def open_tunnel(host, port):
                 "-L",
                 "{}:localhost:{}".format(port, port),
             ]
-            logging.info("Opening tunnel to {}.".format(" ".join(command)))
+            print("Opening tunnel to {}.".format(" ".join(command)))
```

Comment:
Was there some specific issues with `logging` to change to `print`?

Reply:
The gunicorn.conf.py seems to be loaded before the satsale.py, so logging within the ssh_tunnel funcs weren't being written.

Also see https://github.com/SatSale/SatSale/commit/0e773d27ced33b1509e00c7cc8a8a8695f95b9ff which seems to have fixed a similar issue with modules being loaded before logging